wishlist.css

```diff
@@ -0,0 +1,58 @@
+@import url(/global-styles.css);
+
+.main{
+    
+    background-color: var(--off-white);
+    height: 100vh;
+    padding-top: 4rem;
+}
+.wishlist-title{
+    text-align: center;
+    font-size: 1.5rem;
+    font-weight: 600;
+    padding: 1rem;
+}
+
+.main-body{
+    display: flex;
```

Comment:
You could've created utility class just for `display: flex`, would've made it easier to use everywhere. e.g., 
```.disp-flex {display: flex;}```